dockstore-event-consumer/src/main/java/io/dockstore/consumer/handler/DOIHandler.java

```diff
@@ -127,7 +129,9 @@ public boolean handleMessage(DOIMessage message) {
                 // upload a new file
                 int depositionID = returnDeposit.getId();
                 FilesApi filesApi = new FilesApi(zenodoClient);
-                for (SourceFile file : tag.getSourceFiles()) {
+                // This code isn't used currently, so changing it to get around @JsonIgnore being added to sourcefiles. Previously iterated through tag.getSourcefiles()
```

Comment:
I don't understand this. Why do you have a loop on an always empty array?

Do you expect this code to be turned back on? If so, should there be a TODO? If not, shouldn't you just delete it?

Reply:
I think Denis said this was a concept for how you could hook up something like zenodo to dockstore, but Walt went with a different approach. I don't know if it ever be used, but figured it would be good to keep as a reference for possible future work. 
I'll comment this section out instead of changing it.